edenai_apis/apis/google/google_image_api.py

```
@@ -161,6 +163,51 @@ def image__object_detection(
             standardized_response=ObjectDetectionDataClass(items=items),
         )
 
+    async def image__aobject_detection(
+        self, file: str, model: str = None, file_url: str = "", **kwargs
```

Ruff (0.14.4)

Unused method argument: `model`

(ARG002)